packages/dialtone-vue2/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -417,14 +428,15 @@ export default {
         await this.$nextTick();
         const input = this.getInput();
         this.revertInputPadding(input);
-        this.originalInputSize = input.getBoundingClientRect().height;
+        this.initialInputSize = input.getBoundingClientRect().height;
```

Comment:
I think it would still collapse to a single line, because that's the example from storybook (we are loading it with many items already selected). The chips are not really inside the input, but next to it, and the code measures the height of the chips to then add a padding to the input in the method `setInputPadding`, which is called after calculating the initial input height.